### ISSUE TITLE
SSDlite model fails to convert

### ISSUE THREAD

**Issue by Tigrex161** [tf-coreml/tf-coreml#365]
## ❓Question
I used my own training data to perform transfer learning on http://download.tensorflow.org/models/object_detection/ssdlite_mobilenet_v2_coco_2018_05_09.tar.gz
this pretrained ssdlite model. 

The model performs as expected after training. I only changed the number of classes to 1 in the pipeline.config file before training.

Converting the model to tflite format was also successful. Now i need it to run on iOS as well but when trying to convert(even after removing the excess parts of the graph using (https://github.com/apple/coremltools/blob/master/examples/neural_network_inference/tensorflow_converter/Tensorflow_1/ssd_example.ipynb) I run into the following error: 

![error](https://user-images.githubusercontent.com/13764071/70139850-33aba500-16b5-11ea-8cdd-1f7969b87d42.PNG)
 

I inspected the graph using netron and I'm attaching the pictures here. 
![network1](https://user-images.githubusercontent.com/13764071/70139460-5e492e00-16b4-11ea-863b-6ac40ae7dc99.PNG)
![network2](https://user-images.githubusercontent.com/13764071/70139473-63a67880-16b4-11ea-83c8-5ff9d26e865d.PNG)

The batchnorm node causing the error is "FeatureExtractor/MobilenetV2/Conv/BatchNorm/FusedBatchNormV3"
and it does indeed have just one output when inspected. Is it actually supposed to have 5 outputs? What am I doing wrong?

## System Information
- If applicable
Running the code on google colab with: 
TF 1.15
coremltools-3.1 
tfcoreml-1.1

**Comment by aseemw** [comment]
duplicate of issue #362
